CODE_REVIEW.md

```diff
@@ -0,0 +1,65 @@
+# Code reviews and PR process
+
+We use PR reviews to achieve the following purposes:
+  1. Quality: Increase the quality of the software and, as a consequence, of our product
+  2. Training/Coaching/Mentoring: we improve reviewed and reviewers knowledge and skills
+  3. Knowledge sharing: we align the team about technical backgrounds and information
+  4. Idea gatering: we discuss, share and evaluate ideas that creates opportunities
+
+As a developer you are responsible for the quality of your PR and for the amount of time the PR stays open.
+Your PR needs to pass all the reviews steps as fast as possible, you have to communicate and urge the reviewers everytime the PR needs to be reviewed.
+You are invited to read carefully your PR before request a review, this can drastically reduce the time your PR stays open before being approved.
+Reading multiple times your PR changes in diff tools like github can help you spot parts that can be improved.
+
+## PR steps
+### Open a PR
+
+Be sure you are providing the necessary information to speed-up the PR process an to help reviewers doing their part:
+
+* Title should contain the reference to the story/ticket if existing
+* Title should summarise the purpose of the implementation by the code POV, not replicate the title of the story that represent usually the user POV
+* Description
+  1. Should contain link to useful references or resources if available (Jira ticket or tickets, libraries, external documentation...)
+  2. You should provide the right context and background information to help reviewers undestand your implemntation choices, avoided un-necessary subsequent questions ('I used xxx because of xyz...')
+  3. Tests plan: You should provide a basic tests plan. This is to communicate to reviewers that you knew how the feature is expected to work and can also help human QA.
+
+### Assign reviewers
+
+PRs need to be **'peer approved'** and **'core approved'**.
+A peer reviewer is a developer with (very approssimatively) a similar amount of skills and the same seniority and knowledge of the product.
+A core reviewer is a developer with more Seniority and a bigger knowledge of the product and of the 'big picture', usually with also wider background of the business requirements.
+
+Once you are ready, you should assign a **peer reviewer** and contact him/her directly or mentioning in the appropriate PR channel.
```

Comment:
Curious how this has worked for your team. We've posted them in slack and allowed anyone to pick them up, but this causes PRs to sit around, so it's not ideal. I prefer this approach, wondering if you've experienced any problems with it.

Reply:
The idea is to create responsibility for the specific story on a specific person since the beginning till the deploymente. And this responsibility should always be on the developer in the way that 'The story is in review' is acceptable only in one stand up, othewise there is a problem. If the story takes more to be reviewed it's the responsibility of the dev owner to manage this pushing reviewers/change reviewers or rising a problem during the retro about the review process.

Reply:
One other thing that has worked for us well in the past is in Jira the In Review column can easily point out on the jira board for how long that ticket has been in review in a single view. This makes the scrummaster the owner to make sure it is followed with the devs ?

Reply:
Cool, all good points. I'll think ensuring the responsibility of review is on the developer is a good mindset we need to be better about on the CP. Thanks for the feedback guys :-)